tests/functional/export_v2/cram/metadata-with-accession.t

```diff
@@ -0,0 +1,33 @@
+Setup
+
+  $ source "$TESTDIR"/_setup.sh
+
+Run export with metadata that contains "accession".
+
+  $ ${AUGUR} export v2 \
+  >  --tree "$TESTDIR/../data/tree.nwk" \
+  >  --metadata "$TESTDIR/../data/dataset1_metadata_with_strain_and_accession.tsv" \
+  >  --node-data "$TESTDIR/../data/div_node-data.json" "$TESTDIR/../data/location_node-data.json" \
+  >  --auspice-config "$TESTDIR/../data/auspice_config1.json" \
+  >  --maintainers "Nextstrain Team" \
+  >  --output dataset.json > /dev/null
+
+  $ python3 "$TESTDIR/../../../../scripts/diff_jsons.py" "$TESTDIR/../data/dataset_with_accession.json" dataset.json \
+  >   --exclude-paths "root['meta']['updated']" "root['meta']['maintainers']"
+  {}
+
+Run export with metadata that contains "accession", and use "accession" as the ID column.
+Currently, this results in losing the accession from the node attributes.
+
+  $ ${AUGUR} export v2 \
+  >  --tree "$TESTDIR/../data/tree.nwk" \
```

Comment:
I think you may need to add a new `tree.nwk` for this test. The [current tree.nwk](https://github.com/nextstrain/augur/blob/dd195c6a94321a00966aab503197424b5136b1f4/tests/functional/export_v2/data/tree.nwk) uses the `strain` values as the node names. 

This test currently works because of the bug described in https://github.com/nextstrain/augur/issues/1262

Reply:
Thanks for catching this! I actually had to add several different accession-oriented files. I also bundled your fix for #1262 in this PR since it's necessary to make the tests work properly.